google/cloud/forseti/services/scanner/dao.py

```diff
@@ -190,3 +199,34 @@ def map_by_resource(violation_rows):
             v_by_type[v_resource].append(v_data)
 
     return dict(v_by_type)
+
+def _create_violation_hash(violation, inventory_index_id,
```

Comment:
Per description of issue #1059, can you use ```inventory_data, violation data, full_name``` to create the hash?  This was something that andrew, joe, and i `hashed out` to create the uniqueness that we want.

Reply:
Done!